src/Css/Global.elm

```diff
@@ -766,7 +689,7 @@ dl =
 
 {-| Selector for a dt element.
 
-    https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt
+    https :// developer.mozilla.org / en - US / docs / Web / HTML / Element / dt
```

Comment:
Some white spaces seem to have snuck into the Urls

Reply:
Fixed, thanks!